app/sources/AppUtilities.m

```diff
@@ -89,3 +89,8 @@ BOOL parseNumericStringWithSuffix(NSString *stringValue, unsigned long long *res
 invalidString:;
     return NO;
 }
+
+extern int sandbox_check(pid_t, const char *, int);
+BOOL isSandboxed(void) {
+    return !! sandbox_check(getpid(), NULL, 0);
```

Comment:
Private API? Could also just have a #define for the MAS configs and check for that too.

Reply:
Bleh, I suppose we should. It's not quite right because the app may be sandboxed outside of MAS distribution, and also the app may be unsandboxed after the fact manually by the user. Still I guess this API isn't technically public.

Reply:
Up to you. I'm surprised they accepted it into the MAS using a private API though, maybe it's on their whitelist.